guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,32 @@
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
+=== Custom repository sets switching to disabled-by-default
+Custom products are now disabled by default in Katello. During upgrade to Katello 4.9, content overrides will be automatically added so that existing hosts and activation keys have access to the same repositories before and after upgrade. Going forward, newly created custom products will be disabled by default.
```

Comment:
Can we follow one sentence per line here? @maximiliankolb @Lennonka

Reply:
Ideally yes.

Reply:
Yes, it's always a good idea. BUT I don't use this in downstream yet & it's not within `guides/common/`, so I am really OK either way. If the text comes from, say, git commit messages, then I think we should not force anyone to rewrite it.

Reply:
Ack!